glean-core/python/glean/net/deletion_ping_upload_worker.py

```diff
@@ -0,0 +1,95 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+import logging
+from pathlib import Path
+import re
+
+
+log = logging.getLogger(__name__)
+
+
+class DeletionPingUploadWorker:
```

Comment:
It seems like we could remove a lot of code duplication here by just making this a subclass of `PingUploadWorker` and overriding `_PINGS_DIR`, and adding the `try/catch` for `FileNotFoundError` below.

Reply:
Would you let this through if we tackle the deduplication in a follow-up, similar to https://bugzilla.mozilla.org/show_bug.cgi?id=1599427?

Reply:
Sure.